### DISCUSSION TITLE
Improving UI Issue writing in Github

### DISCUSSION THREAD

**Discussion by BenteBak** [community/community#45237] [Projects and Issues]
### 
I am a designer creating issues for developers in Github. I find the toggle between the edit and preview mode exhausting. I would love to just see the code transform into the preview while I type, now when the issue is complex, I have to endlessly scroll and toggle 15 times per issue. 

Also inserting images and not being able to edit the size feels random. Me and my fellow designer have to either accept it, or hack our way into resizing the images. We need to make our issues as scannable and clear as we can for the devs, so they can code faster and little miscommunication happens. 

As a designer ;) I would also suggest some research how users write issues would help enormously in a better UX. I would eb very open to help you improve your product, as I spend a lot of my time on it.

**Comment by canaleal** [comment]
Hey @BenteBak 

I 100% agree with the write and preview at the same time functionality. Still, if you would like to write your issues and see them at the same time, you could use a tool like [Live Markdown Preview](https://markdownlivepreview.com/).

# Image Scaling
 
You can resize the images by using an html image tag and setting the width and height as follows:

### 100x100 image
```
<img src="https://user-images.githubusercontent.com/49886351/214502044-6ddd5ce4-a2ce-4f2f-8f03-36ee135de69c.png" width="100" height="100">
```
<img src="https://user-images.githubusercontent.com/49886351/214502044-6ddd5ce4-a2ce-4f2f-8f03-36ee135de69c.png" width="100" height="100">


### 200x100 image
```
<img src="https://user-images.githubusercontent.com/49886351/214502044-6ddd5ce4-a2ce-4f2f-8f03-36ee135de69c.png" width="200" height="100">
```
<img src="https://user-images.githubusercontent.com/49886351/214502044-6ddd5ce4-a2ce-4f2f-8f03-36ee135de69c.png" width="200" height="100">


### Autox100 image
You can set the width to auto to maintain the aspect ratio of the image. 
```
<img src="https://user-images.githubusercontent.com/49886351/214502044-6ddd5ce4-a2ce-4f2f-8f03-36ee135de69c.png" width="auto" height="100">
```
<img src="https://user-images.githubusercontent.com/49886351/214502044-6ddd5ce4-a2ce-4f2f-8f03-36ee135de69c.png" width="auto" height="100">

**Reply by BenteBak** [reply]
Thanks Alex for your answers. I will give them a try. 

To be honest, all these ideas are still not as intuitive as I am used to having them as a designer. Figma makes it easy to adjust based on a visual aspect and not on a indirect code based understanding. I would say my suggestions will lead to higher ease and satisfaction of using the product. And I am hoping this will be and is considered at high priority, as it affects my daily work a lot, and I am wondering how many other designers and/or non code say persons, like myself, as well.
